dataset_gen/src/camera_controls/scripts/driver_velocity_based.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
+import roslib
+import rospy
+import sys
+import time
+from std_msgs.msg import String
+from camera_controls.msg import msg_transposition
+from tf.transformations import quaternion_from_euler, euler_from_quaternion, quaternion_multiply, quaternion_conjugate, \
+    unit_vector
+from gazebo_msgs.msg import ModelState
+from geometry_msgs.msg import Quaternion, Point
+from gazebo_msgs.srv import SetModelState, GetModelState, GetWorldProperties
+
+
+# Класс сообщения, которое получает драйвер
+class Message:
```

Comment:
As I see it's not the message. It look more familiar to something called "RobotState"